modules/profile/manifests/ros/repo.pp

```diff
@@ -210,7 +210,7 @@
     exec {'init_main_repo':
       path        => '/bin:/usr/bin',
       command     => "/home/${agent_username}/reprepro-updater/scripts/setup_repo.py ubuntu_main -c",
-      environment => ["PYTHONPATH=/home/${agent_username}/reprepro-updater/src:\$PYTHONPATH"],
+      environment => ["PYTHONPATH=/home/${agent_username}/reprepro-updater/src"],
       user  => $agent_username,
       group  => $agent_username,
       unless      => "/home/${agent_username}/reprepro-updater/scripts/setup_repo.py ubuntu_testing -q",
```

Comment:
There is a typo in line 216, it should be:
```
       unless      => "/home/${agent_username}/reprepro-updater/scripts/setup_repo.py ubuntu_main -q",
```

Note ubuntu_main vs ubuntu_testing.

Reply:
Thanks for the catch!